app/assets/javascripts/preferences/panes/AccountPreferences.tsx

```diff
@@ -2,10 +2,15 @@ import { Sync } from '@/preferences/panes/account';
 import { PreferencesPane } from '@/preferences/components';
 import { observer } from 'mobx-react-lite';
 import { WebApplication } from '@/ui_models/application';
+import Credentials from '@/preferences/panes/account/Credentials';
 
-export const AccountPreferences = observer(({application}: {application: WebApplication}) => {
+type Props = {
+  application: WebApplication;
+}
+export const AccountPreferences = observer(({application}: Props) => {
   return (
     <PreferencesPane>
+      <Credentials application={application} />
       <Sync application={application} />
     </PreferencesPane>
   );
```

Comment:
This way it's easier to take everything what we need later on in child components rather than passing each prop separately and possibly having large amount of props in parent component just because one of its children needs some of those props and another child needs other group of props.
P.S. Yeah, I remember that long discussion in Twist :D

Reply:
Sure. I can only give advice.